pages/common/docker.md

```diff
@@ -18,3 +18,11 @@
 - Stop a container
 
 `docker stop {{container}}`
+
+- List all images
+
+`docker images`
+
+- Get a shell inside of an image
```

Comment:
Actually it is correct since he didn't mention container at all, maybe a wording issue. Changing "inside of" to "with" may work better :)

(I also mentioned `docker exec` though :P)